src/router/router.rs

```diff
@@ -159,23 +159,31 @@ fn creates_valid_regex_for_routes () {
     assert_eq!(regex1.is_match("foo/4711/bar?foo=true&bar=false"), false);
     assert_eq!(regex1.is_match("foo/4711/bar/test%20spacing"), true);
     assert_eq!(regex1.is_match("foo/4711/bar/5281?foo=test%20spacing&bar=false"), true);
+    assert_eq!(regex1.is_match("foo/alice/bar/bob"), true);
 
     assert_eq!(regex2.is_match("foo/4711/bar"), true);
     assert_eq!(regex2.is_match("foo/4711/barr"), false);
     assert_eq!(regex2.is_match("foo/4711/bar?foo=true&bar=false"), true);
     assert_eq!(regex2.is_match("foo/4711/4712/bar"), false);
     assert_eq!(regex2.is_match("foo/4711/4712/bar?foo=true&bar=false"), false);
+    assert_eq!(regex2.is_match("foo/alice/bar/bob"), false);
+    //assert_eq!(regex1.is_match("foo/alice/bar"), true);
```

Comment:
No problem. ^-^ I'm new to GitHub so I'm not familiar with conventions and
best practises.

I will aim to get this in by the end of Friday.

On 14 April 2015 at 13:17, Ryman <notifications@github.com> wrote:

> In src/router/router.rs
> <https://github.com/nickel-org/nickel.rs/pull/190#discussion_r28321188>:
>
> >
> >      assert_eq!(regex2.is_match("foo/4711/bar"), true);
> >      assert_eq!(regex2.is_match("foo/4711/barr"), false);
> >      assert_eq!(regex2.is_match("foo/4711/bar?foo=true&bar=false"), true);
> >      assert_eq!(regex2.is_match("foo/4711/4712/bar"), false);
> >      assert_eq!(regex2.is_match("foo/4711/4712/bar?foo=true&bar=false"), false);
> > +    assert_eq!(regex2.is_match("foo/alice/bar/bob"), false);
> > +    //assert_eq!(regex1.is_match("foo/alice/bar"), true);
>
> Would be neat if you could!
>
> Also, mind rebasing to drop the merge commit and to adjust the message
> into line with convention
> <https://github.com/ajoslin/conventional-changelog/blob/master/CONVENTIONS.md>
> ?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/nickel-org/nickel.rs/pull/190/files#r28321188>.
>